### PR TITLE
Some autonomous debugging from Quasar

### PR DESCRIPTION
**Pull request by celskeggs**

### CODE REVIEW

File: QuasarHelios2015/src/org/team1540/quasarhelios/AutonomousModeBase.java [hunk 1]
```diff
@@ -10,6 +10,7 @@
 import ccre.holders.TuningContext;
 import ccre.instinct.AutonomousModeOverException;
 import ccre.instinct.InstinctModeModule;
+import ccre.log.Logger;
```
Comment (amackworth) [inline QuasarHelios2015/src/org/team1540/quasarhelios/AutonomousModeBase.java]: Do we need this import?

File: QuasarHelios2015/src/org/team1540/quasarhelios/Clamp.java [hunk 2]
```diff
@@ -60,7 +60,6 @@ public static void setup() {
         EventStatus zeroEncoder = new EventStatus();
         needsAutoCalibration.setFalseWhen(zeroEncoder);
         FloatInputPoll encoder = Igneous.makeEncoder(10, 11, true, zeroEncoder);
-        Igneous.startAuto.send(zeroEncoder);
```
Comment (amackworth) [inline QuasarHelios2015/src/org/team1540/quasarhelios/Clamp.java:63]: ?

Comment (celskeggs) [inline QuasarHelios2015/src/org/team1540/quasarhelios/Clamp.java:63]: We don't always start with it at the top. And even when we do - the autocalibrator is in charge of this.

Comment (amackworth) [inline QuasarHelios2015/src/org/team1540/quasarhelios/Clamp.java:63]: Ah, right.

**Pull request merged**